docs/docs/how-to-build-custom-client/how-to-build-custom-client.md

```diff
@@ -0,0 +1,481 @@
+                                                                            "x": null,
+                                                                            "y": null
+                                                                        },
+                                                                        "dominantColor": "#6883a1"
+                                                                    }
+                                                                },
+                                                                "urlTemplate": "http://localhost:4000/dam/images/4cf3ef037210cc779fe257a70dd8079277e97cac/c87dafdf-d21c-4b1b-a956-9caf20434d2d/crop:SMART/resize:$resizeWidth:$resizeHeight/adventure-alpine-altitude-austria-355241"
+                                                            }
+                                                        },
+                                                        "activeType": "pixelImage"
+                                                    }
+                                                }
+                                            ]
+                                        }
+                                    }
+                                ]
+                            },
+                            "userGroup": "All"
+                        }
+                    ]
+                }
+            }
+        }
+    }
+}
+```
+
+</details>
+
+The data in the `content` field is a so called `RootBlock`, the `Page` document, can have multiple root blocks. Another `RootBlock`, which is not requested in this example would be the `seo` field. This `RootBlock` will contain all the necessary and available SEO data of the page in the block data structure.
+
+The following JSON structure represents a `HeadlineBlock` in the page content. This block is used to display a headline with optional additional text (eyebrow) and a specific headline level.
+
+Example:
+
+```json
+{
+    "key": "9d261397-6d62-4d6e-8934-0b627f4af4e4",
+    "visible": true,
+    "type": "headline",
+    "props": {
+        "eyebrow": "",
+        "headline": {
+            "draftContent": {
+                "blocks": [
+                    {
+                        "key": "8i42j",
+                        "text": "Lorem ipsum",
+                        "type": "unstyled",
+                        "depth": 0,
+                        "inlineStyleRanges": [],
+                        "entityRanges": [],
+                        "data": {}
+                    }
+                ],
+                "entityMap": {}
+            }
+        },
+        "level": "header-one"
+    },
+    "userGroup": "All"
+}
+```
+
+**HeadlineBlock structure**
+
+-   `key`: A unique identifier for the block.
+-   `visible`: A boolean indicating whether the block is visible.
+-   `type`: The type of the block, in this case, "headline".
+-   `props`: An object containing the block-specific data:
```

Comment:
Those are props of the surrounding blocks block, not the headline block

Reply:
Agree, that was the idea, the caption is wrong. Will change this